integration-tests/src/test/java/org/openmrs/module/fhir2/providers/r3/TaskFhirResourceProviderIntegrationTest.java

```diff
@@ -0,0 +1,339 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.fhir2.providers.r3;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.everyItem;
+import static org.hamcrest.Matchers.hasProperty;
+import static org.hamcrest.Matchers.instanceOf;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.notNullValue;
+import static org.hamcrest.Matchers.startsWith;
+
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import java.util.List;
+import java.util.Objects;
+
+import lombok.AccessLevel;
+import lombok.Getter;
+import org.apache.commons.io.IOUtils;
+import org.hl7.fhir.dstu3.model.Bundle;
+import org.hl7.fhir.dstu3.model.Medication;
+import org.hl7.fhir.dstu3.model.Task;
+import org.junit.Before;
+import org.junit.Test;
+import org.openmrs.module.fhir2.BaseFhirIntegrationTest;
+import org.openmrs.module.fhir2.BaseFhirIntegrationTest.FhirMediaTypes;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.mock.web.MockHttpServletResponse;
+
+public class TaskFhirResourceProviderIntegrationTest extends BaseFhirR3IntegrationTest<TaskFhirResourceProvider, Task> {
+	
+	private static final String TASK_UUID = "55616228-dc6d-446f-ab50-4ec711ea9243";
```

Comment:
This needs to correspond to one of the UUIDs for one of the tasks defined in the `TASK_DATA_XML` file. Since this is an integration test rather than a unit test, it needs to actually be able to find the result in the database.